angular-google-analytics/index.d.ts

```diff
@@ -4,171 +4,232 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="angular" />
-/// <reference path="angular-google-analytics-service.d.ts" />
+import * as angular from 'angular';
 
-declare namespace angular.google.analytics {
+declare module 'angular' {
```

Comment:
As I understand - yes. The easiest way it's compare with existed types e.g. [angular-translate](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-translate/index.d.ts), [angulat-ui-bootstrap](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-ui-bootstrap/index.d.ts) or [angular-ui-router](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/angular-ui-router/index.d.ts).